corrscope/renderer.py

```diff
@@ -158,7 +162,9 @@ def divided_height(self):
         return round(self.height / self.res_divisor)
 
     bg_color: str = "#000000"
+    bg_image: str = ""
     init_line_color: str = default_color()
+    init_line_outline_color: str = "#000000"
```

Comment:
call this `global_` instead of `init_`. I'm planning to rename `init_line_color` too.